File: algorithms-and-data-structures/algorithms-and-data-structures-2/data-structures/linear-data-structures/queue/circularQueue.c
	typedef struct{
		int* array;
		int capacidade,quantidade;
	}Fila;
	void set(*Fila struct,int n){
		(*struct).array=malloc(n*sizeof(int));
		(*struct).capacidade=n;
		(*struct).quantidade=0;
	}
	void enqueue(*Fila struct,int n){
		if((*struct).quantidade==(*struct).capacidade){
			return;
		}
		(*struct).array[(*struct).quantidade]=n;
		(*struct).quantidade=(*struct).quantidade+1%(*struct).capacidade;
	}
	int dequeue(*Fila struct){
		if((*struct).quantidade==0){
			return -1;
		}
		(*struct).quantidade--;
		return (*struct).array[(*struct).inicio++];
	}

File: algorithms-and-data-structures/algorithms-and-data-structures-2/data-structures/linear-data-structures/queue/circularQueue.java
	class Queue{
		int[] array;
		int capacidade,quantidade;
		public Queue(int n){
			array=new int[n];
			capacidade=n;
			quantidade=0;
		}
		void inserir(int n){
			if(quantidade==capacidade){
				return;
			}	
			array[quantidade]=n;
			quantidade=quantidade+1%capacidade;
		}
		int remover(){
			if(quantidade==0){
				return -1;
			}
			quantidade--;
			return array[inicio++];	
		}
	}
